charts/corda/templates/flow-worker.yaml

```
@@ -63,7 +63,7 @@ spec:
           httpGet:
             path: /isHealthy
             port: health
-          periodSeconds: 5
+          periodSeconds: 10

```

Still need to set this back to 5...

```suggestion
          periodSeconds: 5
```